src/wcm.cpp

```diff
@@ -1283,18 +1299,11 @@ void WCM::open_page(Plugin *plugin)
 
 void WCM::load_config_files()
 {
-    wordexp_t exp;
-    char *wf_config_file_override = getenv("WAYFIRE_CONFIG_FILE");
-    char *wf_shell_config_file_override = getenv("WF_SHELL_CONFIG_FILE");
+    const char *wf_config_file_override = getenv("WAYFIRE_CONFIG_FILE");
+    const char *wf_shell_config_file_override = getenv("WF_SHELL_CONFIG_FILE");
 
-    if (wf_config_file_override)
-    {
-        wf_config_file = wf_config_file_override;
-    } else
-    {
-        wordexp(WAYFIRE_CONFIG_FILE, &exp, 0);
```

Comment:
The reason we run the string through wordexp is so that e.g. `WAYFIRE_CONFIG_FILE=~/.config/wayfire` will work to find the home directory.